packages/fast-animation/webpack.config.js

```diff
@@ -0,0 +1,98 @@
+const path                          = require('path');
+const BundleAnalyzerPlugin          = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const MiniCSSExtractPlugin          = require('mini-css-extract-plugin');
+const ForkTsCheckerWebpackPlugin    = require('fork-ts-checker-webpack-plugin');
+const HtmlWebpackPlugin             = require('html-webpack-plugin');
+const ModernizrWebpackPlugin        = require('modernizr-webpack-plugin');
+
+const appDir = path.resolve('./client/examples/app');
+const outDir = path.resolve('./www');
+
+const modernizrConfig = {
+    'options': [
+        'setClasses'
+    ]
+}
+
+module.exports = (env, args) => {
```

Comment:
same question as in the other PR, why is this a function and all of the keys etc moved around?

Reply:
To bring consistency to webpack.config.js.  These did not support passing in parameters like mode etc or optimizations for source maps.  I essentially cloned what @nicholasrice did on his initial implementation of sourcemap removal.  It was difficult to compare because every webpack.config has different ordered keys so I felt organizing this alphabetically was more maintainable.

Reply:
On second look, please remove this and do in another PR, our PRs should be consistent with the issue they are solving, this change has nothing to do with fixing the security vulnerability.